week-1-practise/level-3/1-piping.js

```diff
@@ -10,8 +10,6 @@
   As a learning exercise (you wouldn't do this normally), do all of the above in 1 line of code 
 */
 
-var startingValue = 5;
```

Comment:
You are given `startingValue` in the example here and all the functions you need. The exercise is just supposed to show how you can chain function calls together. So you can do something like `add(multiply(startingValue, 2), 10)`. Think about how you would continue from there.